css/style.css

```diff
@@ -355,21 +351,85 @@ text-align: left;
 }
 
 .filter-header::after {
-  content: "-";
+  content: url(/img/minus.svg);
+}
+
+.toggle-filter-clothing:checked + .filter-header::after {
+  content: url(/img/plus.svg);
+}
+
+.toggle-filter-clothing:checked ~ .filter-options {
+  display: block;
+}
+
+.toggle-filter-clothing ~ .filter-options {
+  display: none;
+}
+
+.toggle-filter-eyes:checked + .filter-header::after {
+  content: url(/img/plus.svg);
+}
+  
+.toggle-filter-eyes:checked ~ .filter-options {
+  display: block;
+}
+
+.toggle-filter-eyes ~ .filter-options {
+  display: none;
+}
+
+.toggle-filter-face:checked + .filter-header::after {
+  content: url(/img/plus.svg);
+}
+ 
+.toggle-filter-face:checked ~ .filter-options {
+  display: block;
+}
+
+.toggle-filter-face ~ .filter-options {
+  display: none;
+}
+
+.toggle-filter-skin:checked + .filter-header::after {
+  content: url(/img/plus.svg);
+}
+ 
+.toggle-filter-skin:checked ~ .filter-options {
+  display: block;
+}
+
+.toggle-filter-skin ~ .filter-options {
+  display: none;
+}
+
+.toggle-filter-background:checked + .filter-header::after {
+  content: url(/img/plus.svg);
+}
+ 
+.toggle-filter-background:checked ~ .filter-options {
+  display: block;
 }
 
-.toggle-filter:checked + .filter-header::after {
-content: "+";
+.toggle-filter-background ~ .filter-options {
+  display: none;
 }
 
-.toggle-filter:checked ~ .filter-options {
+.toggle-filter-special:checked + .filter-header::after {
+  content: url(/img/plus.svg);
+}
+ 
+.toggle-filter-special:checked ~ .filter-options {
   display: block;
 }
 
-.toggle-filter ~ .filter-options {
+.toggle-filter-special ~ .filter-options {
   display: none;
 }
 
+.green-checkbox {
+  accent-color: #21E786;  /*  I havent found a way to colour the checkmark separately from the checkbox background( */
+}
+
 .filter-options {
```

Comment:
Nice, it's much better! If you add vertical paddings for filter options and remove that gray border from the filter container, it will look definitely as in the design! 

![image](https://github.com/user-attachments/assets/ecd903cc-7f29-4df4-a4c7-ad914fab73a5)

Reply:
Well done! Thank you!